Python/Product/Cookiecutter/ViewModel/CategorizedViewModel.cs

```diff
@@ -32,7 +32,7 @@ class CategorizedViewModel : INotifyPropertyChanged {
             _displayName = displayName;
         }
 
-        public bool Selectable => false;
+        public bool Selectable => true;
```

Comment:
If you haven't already, check that things work properly when the category is selected. For example, menu/toolbar items disabled as appropriate, template description section at the bottom is correct.